### PR TITLE
feat(search): 2586 missing msg

### PR DESCRIPTION
**Pull request by cy**
- disable 'Add all to cart' when no files
- cleanups to gdc-table

Closes #2586

### CODE REVIEW

**Comment by alex-wilmer** [comment]
👍

File: app/scripts/cart/templates/add-to-cart-all-button.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   id="add-to-cart-button"
   class="btn btn-primary"
   data-ng-click="ctrl.addAll(ctrl.files)"
-  data-ng-disabled="!ctrl.hasFiles"
+  data-ng-disabled="ctrl.size === '0' || (ctrl.files && ctrl.files.length === 0)"
```
Comment (shanewilson) [inline app/scripts/cart/templates/add-to-cart-all-button.html]: `'0'` vs `0` ?

Comment (cy) [inline app/scripts/cart/templates/add-to-cart-all-button.html]: https://github.com/NCI-GDC/portal-ui/pull/1006/commits/1571c3f4e0805bb518dec2edc10ae908af129305#diff-a4de33ae03db0ff7e45bd7fc76f505e1L66 was converting size to a string, updated

**Comment by shanewilson** [comment]
👍

**Comment by ithinkicancode** [comment]
👍

**Pull request merged**